### PR TITLE
Implement aside section

### PR DESCRIPTION
**Pull request by lera-sh**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -35,6 +37,7 @@ div {
 
 .menu-item {
   border-radius: 35px;
+  margin-left: 550px;
```
Comment (katsu223) [inline assets/css/style.css:40]: I think it is redundant

File: assets/css/style.css [hunk 2]
```diff
@@ -52,14 +55,18 @@ div {
...
   align-items: center;
   background-color: #000;
+  border: none;
   border-radius: 9999px;
   box-shadow: 0 40px 50px 0 rgba(0 ,0, 0, .5);
   color: #f0cdab;
   display: flex;
   justify-content: center;
+  margin-left: 450px;
+  margin-top: 500px;
```
Comment (katsu223) [inline assets/css/style.css:65]: Same here

File: assets/css/style.css [hunk 3]
```diff
@@ -69,8 +76,40 @@ div {
   opacity: .8;
 }
 
-.main-section {
+aside {
```
Comment (katsu223) [inline assets/css/style.css:79]: Do not use ```tags``` to style elements

File: assets/css/style.css [hunk 4]
```diff
@@ -69,8 +76,40 @@ div {
...
+  transform: rotate(180deg);
+  width: 120px;
+  writing-mode: tb;
+}
+
+.primary-phrase {
+  color: #000;
+}
+
+.strong-text {
```
Comment (katsu223) [inline assets/css/style.css:101]: Not a good approach. Use something that have semantics

File: assets/css/style.css [hunk 5]
```diff
@@ -69,8 +76,40 @@ div {
...
+  color: #000;
+}
```
Comment (katsu223) [inline assets/css/style.css:103]: In your case, you can add color only for parent element and for primary/secondary phrase

File: assets/css/style.css [hunk 6]
```diff
@@ -69,8 +76,40 @@ div {
...
+
+.main-section {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.icon-bag {
+  margin-bottom: 250px;
```
Comment (katsu223) [inline assets/css/style.css:112]: Better to use ```padding-bottom``` on ```aside``` tag

File: assets/css/style.css (hunk 1 above)
Comment (lera-sh) [reply assets/css/style.css:40]: fixed

File: assets/css/style.css (hunk 2 above)
Comment (lera-sh) [reply assets/css/style.css:65]: fixed

File: assets/css/style.css (hunk 3 above)
Comment (lera-sh) [reply assets/css/style.css:79]: fixed

File: assets/css/style.css (hunk 4 above)
Comment (lera-sh) [reply assets/css/style.css:101]: fixed

File: assets/css/style.css (hunk 6 above)
Comment (lera-sh) [reply assets/css/style.css:112]: fixed

File: assets/css/style.css (hunk 5 above)
Comment (lera-sh) [reply assets/css/style.css:103]: fixed

File: assets/css/style.css (hunk 4 above)
Comment (lera-sh) [reply assets/css/style.css:101]: fixed

File: assets/css/style.css (hunk 3 above)
Comment (lera-sh) [reply assets/css/style.css:79]: fixed

File: assets/css/style.css (hunk 2 above)
Comment (lera-sh) [reply assets/css/style.css:65]: fixed

File: assets/css/style.css (hunk 1 above)
Comment (lera-sh) [reply assets/css/style.css:40]: fixed

File: assets/css/style.css [hunk 7]
```diff
@@ -40,6 +51,69 @@ div {
...
+  height: 100%;
+  width: 100%;
+}
+
+.icon-bag {
+  align-self: flex-end;
+  background-color: #fff;
+  display: flex;
+  justify-content: center;
+  padding: 42.5px;
```
Comment (katsu223) [inline assets/css/style.css:112]: You have ```position: absolute```, why are you using ```padding```?

**Pull request merged**